### DISCUSSION TITLE
How to show the integration_status status in sales invoice list

### DISCUSSION THREAD

**Discussion by BOTSolutions8686** [lavaloon-eg/ksa_compliance#143] [Q&A]
I want to show the integration_status status in sales invoice list. how can I add as it is available in additional fields but i want to show it derectly in sales invoice list. Any idea ?

**Comment by rizwan100mu** [comment]
It's very easy

1. Identify the Field to Fetch From:

Locate the Doctype that contains the integration_status field.

Find the link field that connects this Doctype to the Sales Invoice.



2. Add a Custom Field in Sales Invoice:

Go to Customize Form in ERPNext.

Select Sales Invoice as the Doctype to customize.

Click Add Row to create a new custom field and set the following properties:

Label: Integration Status

Field Type: Data (or appropriate type based on the source field)

Fieldname: integration_status




3. Set the Fetch From Property:

In the new custom field, set the Fetch From attribute.

The format should be linked_doctype_fieldname.integration_status, where:

linked_doctype_fieldname is the name of the link field in Sales Invoice that connects to the other Doctype.

integration_status is the field name in the other Doctype.

**Reply by BOTSolutions8686** [reply]
Does not work. You will have to write a custom client script
